internal/oci/container.go

```diff
@@ -152,6 +160,12 @@ func (c *Container) StopSignal() syscall.Signal {
 }
 
 // FromDisk restores container's state from disk
+// Calls to FromDisk should always be proceeded by call to Runtime.UpdateContainerStatus
```

Comment:
did you mean preceded?

```suggestion
// Calls to FromDisk should always be preceded by call to Runtime.UpdateContainerStatus.
```